geminim.ini

```diff
@@ -7,13 +7,27 @@ dirHeader = "header" # Prepend a directory's listing with the file inside the di
 # Path to the directory containing your vhost public files.
 # For each vhost you should have a directory with the name of the hostname,
 # which will be the root directory of the said vhost.
+# For example, example.com should be at rootDir/example.com
 # Defaults to "pub/"
 rootDir = "pub/"
 
 # Path to a public user directory. $# represents the username
 # Defaults to "/home/$#/" on POSIX systems and "C:\Users\$#" on Windows
 homeDir = "/home/$#/pub"
 
+# Password that the client must provide in order to get write access
+# to files on the server. Defaults to "titanpassword" --
+# FOR OBVIOUS REASONS YOU SHOULD DEFINITELY CHANGE IT
+titanpass = "titanpassword"
```

Comment:
These new options could be written as camelCase to keep consistency with the rest of the config